lib/ansible/modules/windows/win_chocolatey.py

```diff
@@ -189,12 +281,8 @@
 
 - name: uninstall multiple packages
   win_chocolatey:
-    name: '{{ item }}'
+    name: procexp, putty, windirstat
```

Comment:
Should we use proper YAML in the examples, rather than advice people to use a backward-compatible interface (this comes from k=v syntax originally) ?

Reply:
Also, do we need to add **win_chocolatey** to the list of modules that support squashing ?
```
# squash actions
# Ansible can optimise actions that call modules with list parameters
# when looping. Instead of calling the module once per with_ item, the
# module is called once with all items at once. Currently this only works
# under limited circumstances, and only with parameters named 'name'.
#squash_actions = apk,apt,dnf,homebrew,pacman,pkgng,yum,zypper
```

Reply:
I had the proper YAML use earlier and through it best to show the alternative comma separated way but happy to put it back to a list to not encourage k=v syntax.

I personally don't think we want to add to the list of modules that support squashing considering non squash was the default behaviour and now people can choose either all and not be at the mercy of a global config.